glustercli/cmd/snapshot-info.go

```diff
@@ -47,6 +47,7 @@ func snapshotInfoDisplay(snap api.SnapGetResp) {
 	fmt.Println("Snapshot Volume ID:", vol.ID)
 	fmt.Println("State:", vol.State)
 	fmt.Println("Origin Volume name:", snap.ParentVolName)
+	fmt.Println("Snapshot Description:", snap.Description)
```

Comment:
A long description will be awful to print in the middle. Can you please move to the end. If you could limit the total strdout content in a line to 80 characters, that will be great